Jenkinsfile

```diff
@@ -1,18 +1,26 @@
 #!groovy
-node('xnetPython01') {
+node('xnetPython') {
 
 	currentBuild.result = "SUCCESS"
 	
 	try{
-		stage('Test'){
-			// test script to checkout the scm
+		stage('Checkout'){
+			// Checkout the repository from scm
+			echo "Cheking out source"
 			checkout scm
 		}		
+		
+		stage('Testing'){
+			// Run tox with the tox-integration.ini file in the root of the repository
+			echo "Running Tox integration script"
+			bat 'tox -c tox-integration.ini'
```

Comment:
So we don't forget to later, we should set the environment variables needed by `tox-integration.ini`.  They are defaulted for ease-of-use for contributors but we should explicitly map them on the test node to make sure we are getting our desired results.